### PR TITLE
UNO: make uno less subtle, allow player to uno before next player plays.

### PR DESCRIPTION
**Pull request by sparkychildcharlie**
- Make UNO button be there every turn, so it's less of a "reminder" for the players to hit when it comes out
- fix some HTML escaping I had left out previously
- add a 0.75 second delay for the uno button to get pressed
    - (chat filter uses up 600 ms, and the rest (150) can be used to process.  The command will be queue'd regardless so it will run after the 600 mark)
    - make it fairer for players to get UNO in, and to make UNO bots not broken af when it comes to sniping users at UNO

### CODE REVIEW

File: chat-plugins/uno.js [hunk 1]
```diff
@@ -483,7 +502,7 @@ class UNOgamePlayer extends Rooms.RoomGamePlayer {
 		this.sendRoom(
 			`|uhtml|uno-hand|<div style="border: 1px solid skyblue; padding: 0 0 5px 0"><table style="width: 100%; table-layout: fixed; border-radius: 3px"><tr><td colspan=4 rowspan=2 style="padding: 5px"><div style="overflow-x: auto; white-space: nowrap; width: 100%">${hand}</div></td>${this.game.currentPlayer === this.userid ? `<td colspan=2 style="padding: 5px 5px 0 5px">${top}</td></tr>` : ""}` +
 			`<tr><td colspan=2 style="vertical-align: top; padding: 0px 5px 5px 5px"><div style="overflow-y: scroll">${players}</div></td></tr></table>` +
-			`${this.game.currentPlayer === this.userid ? `<div style="text-align: center">${draw}<span style="padding: 15px"></span>${pass}</div>` : ""}</div>`
+			`${this.game.currentPlayer === this.userid ? `<div style="text-align: center">${draw}<span style="padding: 8px"></span>${pass}<span style="padding: 8px"></span>${uno}</div>` : ""}</div>`
```
Comment (panpawn) [inline chat-plugins/uno.js]: What is the second ``<span>`` tag doing here, exactly?

Comment (sparkychildcharlie) [inline chat-plugins/uno.js]: it's a spacing thing - basically just a space of 8 pixels instead of a string of ``&nbsp;``s

Comment (Zarel) [inline chat-plugins/uno.js]: Should do `padding-left:16px` or something.

Comment (Zarel) [inline chat-plugins/uno.js]: Should also probably add a space before or after, for semantic reasons

Comment (sparkychildcharlie) [inline chat-plugins/uno.js]: done @Zarel 
i hope i understood you properly here :x

**Comment by Zarel** [comment]
Honestly, though, I think that the time limit for "Uno" should be different from the time it takes the next person to move. Calling someone on not Unoing should also be done manually rather than automatically.

**Comment by sparkychildcharlie** [comment]
Hm... that may be a good point.   The rules for this are rather ambiguous since it's difficult/impossible to really find the latest "official" guide, and everyone plays with some differing quirks to the game

**Pull request merged**